### ISSUE TITLE
configure error in release 2017.04.03.00

### ISSUE THREAD

**Issue by set-daemon** [facebook/proxygen#164]
after executed "autoreconf -ivf", it generates file configure, in which a line "AX_PREFIX_CONFIG_H(proxygen-config.h, proxygen, config.h)" seems to be generated wrongly. 
so i delete it and execute "./configure" again, it works successfully.
i'm wondering whether anyone else has encountered the same problem?

**Comment by dddmello** [comment]
Tried to reproduce but unable too.

./deps.sh calls 'autoreconf -ivf' and './configre' right after it every time before building proxygen.  I tried running the command manually as well and the configure file that was generated looks correct and does not appear to contain any line resembling the above.  I was testing on ubuntu 16.04.2.

If there is more information are you have more details to help with a repro let me know.
